### ISSUE TITLE
[standard] Commented snippets are all over the template we need to make it clean

### ISSUE THREAD

**Issue by fusionx1** [hechoendrupal/DrupalConsole#1377]
Hi im Paul from Drupal Pilipinas and i really want to contribute to this wonderful project that helps and will help thousands of drupal developer around the world.

Motivation: 
 
As i was doing a cursory check on the some of the files ive noticed that there are lots of code snippets that are not being used, which i think it should be removed if we are not using it, especially inside the twig file.  I'm really a strong believer of drupal coding standard big or small we should embrace it. 

Solution:

I know those commented code snippets were there for a reason, i recommend to remove it in the template file and put it somewhere probably a seperte file that inventory all the functions and if time comes we can retrieve it from that file.

Heres a format:

#snippet 1:

//https://github.com/hechoendrupal/DrupalConsole/blob/master/templates/theme/theme.twig
//line 54 
//hook_theme_suggestions_node_alter

/*$node = $variables['elements']['#node'];

  if ($variables['elements']['#view_mode'] == "full") {

  }*/

**Comment by fusionx1** [comment]
related issue https://github.com/hechoendrupal/DrupalConsole/issues/693

**Comment by jmolivas** [comment]
@fusionx1: I totally agree on removing commented code. I did remove several of those in past PR.

We are planning to provide a `--learning` option when executing the project, we may include some of that code when the project is been executed using passing that option.
